albumentations/augmentations/functional.py

```diff
@@ -2041,3 +2041,48 @@ def glass_blur(img, sigma, max_delta, iterations, dxy, mode):
             x[h, w], x[h + dy, w + dx] = x[h + dy, w + dx], x[h, w]
 
     return np.clip(cv2.GaussianBlur(x / coef, sigmaX=sigma, ksize=(0, 0)), 0, 1) * coef
+
+
+def fourier_domain_adaptation(img, target_img, beta):
+    """
+    Fourier Domain Adaptation from https://github.com/YanchaoYang/FDA
+
+    Args:
+        img:  source image
+        target_img:  target image for domain adaptation
+        beta: coefficient from source paper
+
+    Returns:
+        transformed image
+
+    """
+    # get fft of both source and target
+    fft_src = np.fft.fft2(img.astype(np.float32), axes=(0, 1))
+    fft_trg = np.fft.fft2(target_img.astype(np.float32), axes=(0, 1))
+
+    # extract amplitude and phase of both fft-s
+    amplitude_src, phase_src = np.abs(fft_src), np.angle(fft_src)
+    amplitude_trg = np.abs(fft_trg)
+
+    # mutate the amplitude part of source with target
+    amplitude_src_shifted = np.fft.fftshift(amplitude_src, axes=(0, 1))
+    amplitude_trg_shifted = np.fft.fftshift(amplitude_trg, axes=(0, 1))
+
+    height, width, _ = amplitude_src_shifted.shape
+    border = (np.floor(np.min((height, width)) * beta)).astype(int)
```

Comment:
We can use there built in `min` function, I think this code a little bit clearly:
```python
np.floor(min(height, width) * beta).astype(int)
```

Reply:
done.